nemo_skills/pipeline/start_server.py

```diff
@@ -20,6 +20,16 @@
 from nemo_skills.utils import setup_logging
 
 
+def get_gradio_chat_cmd(server_type, with_sandbox, extra_args):
+    cmd = (
+        "python -m nemo_skills.inference.launch_chat_interface "
+        f"   --server_type {server_type} "
+        f"   --with_sandbox {with_sandbox} "
+        f"   {extra_args} "
```

Comment:
as discussed offline, let's control all arguments from web interface itself, not here